include/jemalloc/internal/tsd.h

```diff
@@ -72,7 +72,8 @@ typedef ql_elm(tsd_t) tsd_link_t;
     O(binshards,		tsd_binshards_t,	tsd_binshards_t)\
     O(tsd_link,			tsd_link_t,		tsd_link_t)	\
     O(in_hook,			bool,			bool)		\
-    O(tcache_slow,		tcache_slow_t,		tcache_slow_t)
+    O(tcache_slow,		tcache_slow_t,		tcache_slow_t)	\
+    O(rtree_ctx,		rtree_ctx_t,		rtree_ctx_t)
```

Comment:
I understand the point, but this'd better need some more comments, as it contradicts your definition.

Reply:
Updated comments.